10-Types-of-contributions.md

```diff
@@ -0,0 +1,228 @@
+# Types of contributions
+In this chapter, we will discuss the types of contributions that open-source projects involve.
+
+Mostly, there are two types of contributions. Which are;
+
+- Non-coding type of contribution
+
+- Coding type of contribution
+
+   ## A. Non-Coding Type Of Contributions
+  
+  Non-coding contributions do not involve writing or modifying code. Non-coding contributions are essential for the sustainability and growth of open-source projects. They allow individuals with a variety of skills and backgrounds to participate and make meaningful contributions to the software and the community.
+
+ ### Common types of non-coding contributions
+ 
+  - Documentation
+  - Graphic design
+  - Blog writing
+
+
+### Documentation
+
+Are you a lover of words? Do grammar errors irritate you, like seeing pineapple on pizza? Then, documentation is for you! 
+This involves creating, improving, or maintaining the written resources that accompany open-source software projects. Contrary to popular belief, documentation is crucial to open-source projects. It can make a project more accessible and user-friendly, and it can help attract new contributors.
+
+### How to Contribute to Documentation
+
+*If you're interested in contributing to documentation, here are a few tips:*
+
+- **Read the documentation**: This will help you understand how the product is used.
+  
+- **Find a task that's a good fit for your skills and interests**: If you're not sure where to start, ask the project's maintainers for suggestions.
```

Comment:
I would recommend looking at the issues or discussions before reaching out to a maintainer.